fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -84,8 +84,13 @@
     // FIXITY
 
     public static final Resource FIXITY_TYPE = createResource(PREMIS_NAMESPACE + "Fixity");
+
+    public static final Property HAS_MESSAGE_DIGEST_ALGORITHM =
+            createProperty(PREMIS_NAMESPACE + "hasMessageDigestAlgorithm");
```

Comment:
FYI, the current authoritative PREMIS ontology is http://id.loc.gov/ontologies/premis.rdf (linked to from the bottom of the HTML page @claussni linked to) and the current version of the data dictionary reference is http://www.loc.gov/standards/premis/v3/premis-3-0-final.pdf.